fiftyone/utils/sam2.py

```
@@ -602,12 +643,16 @@ def load_fiftyone_video_frames(
 
     num_frames = len(sample.frames)
     try:
-        images = torch.zeros(num_frames, 3, image_size, image_size, dtype=torch.float32)
+        images = torch.zeros(
+            num_frames, 3, image_size, image_size, dtype=torch.float32
+        )
     except Exception as e:
         raise (e)
```

Ruff (0.15.12)

[warning] Remove exception handler; error is immediately re-raised

(TRY203)